packages/tracking/src/integrations/__tests__/conditionallyLoadAnalytics-test.ts

```diff
@@ -67,12 +75,32 @@ describe('conditionallyLoadAnalytics', () => {
       id: 'abc123',
     };
     const options = createMockOptions({ user });
+    mockClientType.mockReturnValue('default');
+
+    conditionallyLoadAnalytics(options);
+
+    expect(options.analytics.identify).toHaveBeenCalledWith(
+      user.id,
+      { email: user.email, client: 'default' },
+      {
+        integrations: options.identifyPreferences,
+      }
+    );
+  });
+
+  it('sets source to PWA on analytics.identify when source is a PWA', () => {
```

Comment:
```suggestion
  it('sets client to PWA on analytics.identify when source is a PWA', () => {
```

_(and the other one?)_